omoefe-dukuye/SendIT
# Ch set up code coverage 161795309

#### What this PR does
Set up code coverage and display README badge using [Coveralls.io](https://coveralls.io).
(Finishes [#161795309](https://www.pivotaltracker.com/story/show/161795309))

#### Description of Tasks
- Add repo on [Coveralls.io](https://coveralls.io)
- Install coveralls
- Install mocha-lcov-reporter
- Install nyc
- Adjust package.json scripts
- Add readme badge

#### How to test manually
- Clone the repo `git clone https://github.com/un-tethered/SendIT.git`
- Checkout to the branch `git checkout ch-set-up-code-coverage-161795309`
- Run `npm install` to install application dependencies
- Test by runnung `npm test`
You should see coverage results
- See report [here](https://coveralls.io/github/un-tethered/SendIT?branch=develop&service=github)


#### Pivotal tracker story
[#161795309](https://www.pivotaltracker.com/story/show/161795309)

#### Screeshot
[![Coverage Status](https://coveralls.io/repos/github/un-tethered/SendIT/badge.svg?branch=develop&service=github)](https://coveralls.io/github/un-tethered/SendIT?branch=develop&service=github)
